.github/workflows/pr.yml

```
@@ -0,0 +1,29 @@
+name: PR
+on:
+  pull_request:
+    paths-ignore:
+      - ".github/workflows/**"
+jobs:
+  pr:
+    name: pr
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)